{"name": "test_getbyid_failure", "status": "passed", "start": 1729274488688, "stop": 1729274488688, "uuid": "da52f773-0bf8-4c7c-9258-5b966b708d40", "historyId": "cbc29a25fe64c1c9081b3685cd78d352", "testCaseId": "cbc29a25fe64c1c9081b3685cd78d352", "fullName": "Exercise.exercise_test.TestExerciseServiceWithoutMock#test_getbyid_failure", "labels": [{"name": "parentSuite", "value": "Exercise"}, {"name": "suite", "value": "exercise_test"}, {"name": "subSuite", "value": "TestExerciseServiceWithoutMock"}, {"name": "host", "value": "DESKTOP-IDGJ8R7"}, {"name": "thread", "value": "7844-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Exercise.exercise_test"}]}